NYUDining/EditProfileViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  EditProfileViewController.swift
+//  NYUDining
+//
+//  Created by Jesus Leal on 6/20/16.
+//  Copyright © 2016 Ross Freeman. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import Alamofire
+
+
+class EditProfileViewController: UIViewController {
+    
+    @IBOutlet weak var schoolPicker: UIPickerView!
+    
+    @IBOutlet weak var year: UITextField!
+    
+    @IBOutlet weak var likes: UITextView!
+    
+    @IBOutlet weak var major: UITextField!
+    
+    var schoolDataSource : Array = ["College of Arts & Science", "Graduate School of Arts and Science", "College of Dentistry", "College of Global Public Health", "Courant Institute", "Gallatin School", "Institute of Fine Arts", "Institute for the Study of the Ancient World", "Leonard N. Stern School of Business", "Robert F. Wagner Graduate School of Public Service", "Rory Meyers College of Nursing", "School of Professional Studies", "School of Law", "School of Medicine", "Silver School of Social Work", "Steinhardt School of Culture, Education, and Human Development", " Tandon School of Engineering", "Tisch School of the Arts", "NYU Abu Dhabi", "NYU Shanghai"];
```

Comment:
This would be better suited as an enum or in some sort of tag file that quite literally just holds string constants (so for example you'd just call SchoolNames.Stern)